src/main/native/SymmetricCipher.c

```diff
@@ -449,21 +449,34 @@ JNIEXPORT int CIPHER_zKMC_internal(unsigned char* input, unsigned char* output,
 
 JNIEXPORT jint JNICALL Java_com_ibm_crypto_plus_provider_ock_NativeInterface_z_1kmc_1native(JNIEnv * env, jclass clazz, jbyteArray input, jint inputOffset, jbyteArray output, jint outputOffset, jlong paramPointer, jint inputLength, jint mode)
 {
-  // Get input and output buffer
-  jboolean isCopy = 0;
+  jboolean isCopy = JNI_FALSE;
   jint len = 0;
-  unsigned char* inputPointer  = (unsigned char*)((*env)->GetPrimitiveArrayCritical(env, input,  &isCopy)) + inputOffset;
-  unsigned char* outputPointer = (unsigned char*)((*env)->GetPrimitiveArrayCritical(env, output, &isCopy)) + outputOffset;
 
-  if( NULL == outputPointer || NULL == inputPointer ) {
+  // Get input buffer.
+  unsigned char* inputPointer  = (unsigned char*)((*env)->GetPrimitiveArrayCritical(env, input,  &isCopy));
```

Comment:
All variable definitions should be at the top of the function. Make it easier to find them in a function if the function gets large.

Reply:
Yes this makes sense to do. Updated this code with https://github.com/IBM/OpenJCEPlus/pull/294/commits/c47900e797ce4b47f2bd22cc1d0c1ffd9e994868 to declare `inputPointer` and `outputPointer` at the top of the method.